blockchain/system/auction.go

```diff
@@ -61,3 +60,31 @@ func EncodeAuctionCallData(bid *auction.Bid) ([]byte, error) {
 
 	return data, nil
 }
+
+func DecodeAuctionCallData(encoded []byte) (*auction.Bid, error) {
```

Comment:
Please leave a comment on why this function is required.

Reply:
Appended. Thank you